.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-on: [push, pull_request]
+on: pull_request
```

Comment:
no master tests? I'm fine with, but.. then we don't know the state of master before releasing

Reply:
master tests are run on the other job, which triggers on push!